go.mod

```diff
@@ -5,25 +5,28 @@ go 1.12
 replace github.com/hashicorp/consul => github.com/hashicorp/consul v1.5.1
 
 require (
-	cloud.google.com/go v0.44.1
+	cloud.google.com/go/bigtable v1.1.0
+	cloud.google.com/go/storage v1.3.0
 	github.com/alecthomas/chroma v0.7.0
 	github.com/alecthomas/repr v0.0.0-20181024024818-d37bc2a10ba1 // indirect
-	github.com/cortexproject/cortex v0.4.0
+	github.com/cortexproject/cortex v1.1.0
 	github.com/dlclark/regexp2 v1.2.0 // indirect
-	github.com/jpillora/backoff v1.0.0 // indirect
 	github.com/mattn/go-isatty v0.0.9 // indirect
 	github.com/mitchellh/colorstring v0.0.0-20190213212951-d06e56a500db
-	github.com/opentracing/opentracing-go v1.1.0
-	github.com/pkg/errors v0.8.1
+	github.com/opentracing/opentracing-go v1.1.1-0.20200124165624-2876d2018785
+	github.com/pkg/errors v0.9.1
 	github.com/prometheus/alertmanager v0.20.0
-	github.com/prometheus/client_golang v1.3.0
-	github.com/prometheus/common v0.7.0
-	github.com/prometheus/prometheus v1.8.2-0.20190918104050-8744afdd1ea0
-	github.com/sirupsen/logrus v1.4.2
+	github.com/prometheus/client_golang v1.5.0
+	github.com/prometheus/common v0.9.1
+	github.com/prometheus/prometheus v1.8.2-0.20200213233353-b90be6f32a33
+	github.com/sirupsen/logrus v1.5.0
 	github.com/stretchr/testify v1.4.0
-	google.golang.org/api v0.8.0
+	google.golang.org/api v0.14.0
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
-	gopkg.in/yaml.v2 v2.2.2
+	gopkg.in/yaml.v2 v2.2.8
 	gopkg.in/yaml.v3 v3.0.0-20200506231410-2ff61e1afc86
-	sigs.k8s.io/yaml v1.1.0
 )
+
+replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v36.2.0+incompatible
```

Comment:
Why are these here?

Reply:
Let's add a comment on the why before merging if it's for a special reason

Reply:
These are overrides that are ported over from cortex due to some dependency issues there.